src/components/Timer/index.tsx

```diff
@@ -35,6 +43,10 @@ export default function Question({ duration, running = false, title, handleTimeU
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [running, remainingTime]);
 
+  useEffect(() => {
```

Comment:
Why do we need this?

Reply:
there was a bug in your code.. when timer ends for a question, remaining time does not reset to `duration` value again (as the timer component does not unmounted on timer ends), so `handleTimeUp` was getting called in every second

Reply:
It works fine in production. Something that you have changed which caused this?

Reply:
I saw that bug on master branch. 
I created a game with `10 sec` timeout and then I let the question selection timer finished

Reply:
OK